### PR TITLE
Ts 674

### PR DESCRIPTION
**Pull request by tomskikh**

### CODE REVIEW

File: src/test/scala/com/bwsw/tstreams/benchmark/Benchmark.scala [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    result
+  }
+
+
+  /**
+    * Measures execution time of methods [[com.bwsw.tstreams.agents.consumer.Consumer.getTransactionsFromTo()]] and
+    * [[com.bwsw.tstreams.agents.consumer.ConsumerTransaction.getAll]]
+    *
+    * @param iterations         amount of measurements
+    * @param interval           amount of transaction that retrieves by one invocation of
```
Comment (placccebo) [inline src/test/scala/com/bwsw/tstreams/benchmark/Benchmark.scala]: that retrieves -> retrieved

File: src/test/scala/com/bwsw/tstreams/benchmark/Benchmark.scala [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+    *                           [[com.bwsw.tstreams.agents.consumer.Consumer.getTransactionsFromTo()]]
+    * @param partition          stream partition
+    * @param dataSize           size of data sent in each transaction
```
Comment (placccebo) [inline src/test/scala/com/bwsw/tstreams/benchmark/Benchmark.scala]: in -> to

File: src/test/scala/com/bwsw/tstreams/benchmark/BenchmarkConfig.scala [hunk 3]
```diff
@@ -73,6 +73,9 @@ class BenchmarkConfig(arguments: Seq[String]) extends ScallopConf(arguments) {
   val loadData: ScallopOption[Boolean] = opt[Boolean](
     descr = "if set, data will be retrieved from transactions")
 
+  val interval: ScallopOption[Int] = opt[Int](
+    descr = "amount of transaction that retrieves by one invocation of Consumer.getTransactionsFromTo() (optional)")
```
Comment (placccebo) [inline src/test/scala/com/bwsw/tstreams/benchmark/BenchmarkConfig.scala]: that retrieves -> retrieved

File: src/test/scala/com/bwsw/tstreams/benchmark/ConsumerBenchmarkRunner.scala [hunk 4]
```diff
@@ -32,15 +32,29 @@ package com.bwsw.tstreams.benchmark
   * --data-size - size of data sent in each transaction (100 by default);
   * --partition - stream partition (0 by default);
   * --load-data - if set, data will be retrieved from transactions
+  * --interval - amount of transaction that retrieves by one invocation of (optional)
```
Comment (placccebo) [inline src/test/scala/com/bwsw/tstreams/benchmark/ConsumerBenchmarkRunner.scala]: that retrieves -> retrieved

File: src/test/scala/com/bwsw/tstreams/benchmark/Benchmark.scala (hunk 1 above)
Comment (tomskikh) [reply src/test/scala/com/bwsw/tstreams/benchmark/Benchmark.scala]: fixed

File: src/test/scala/com/bwsw/tstreams/benchmark/Benchmark.scala (hunk 2 above)
Comment (tomskikh) [reply src/test/scala/com/bwsw/tstreams/benchmark/Benchmark.scala]: fixed

File: src/test/scala/com/bwsw/tstreams/benchmark/BenchmarkConfig.scala (hunk 3 above)
Comment (tomskikh) [reply src/test/scala/com/bwsw/tstreams/benchmark/BenchmarkConfig.scala]: fixed

File: src/test/scala/com/bwsw/tstreams/benchmark/ConsumerBenchmarkRunner.scala (hunk 4 above)
Comment (tomskikh) [reply src/test/scala/com/bwsw/tstreams/benchmark/ConsumerBenchmarkRunner.scala]: fixed

**Pull request merged**